README.md

```diff
@@ -370,6 +372,14 @@ Nighthawk process. Note that increasing this results in an effective
 load multiplier combined with the configured --rps and --connections
 values. Default: 1.
 
+--http3-protocol-options <string>
+HTTP3 protocol options in json. If specified, Nighthawk uses these
```

Comment:
Does it make sense to link to the schema here?

Reply:
Agreed that it may be hard to find the protocol options. We don't directly edit this portion of the README, it gets auto-generated from the CLI help. I have updated that help to include the full name of the message including its c++ namespace, which should help when looking for it.

Done in 9d9b697305941adcf4a1eb28cde7f919d79a7859.